sonar-ws/src/main/gen-java/org/sonarqube/ws/QualityProfiles.java

```diff
@@ -0,0 +1,2483 @@
+          java.lang.String s = bs.toStringUtf8();
+          if (bs.isValidUtf8()) {
+            language_ = s;
+          }
+          return s;
+        }
+      }
+      /**
+       * <code>optional string language = 3;</code>
+       */
+      public com.google.protobuf.ByteString
+          getLanguageBytes() {
+        java.lang.Object ref = language_;
+        if (ref instanceof java.lang.String) {
+          com.google.protobuf.ByteString b = 
+              com.google.protobuf.ByteString.copyFromUtf8(
+                  (java.lang.String) ref);
+          language_ = b;
+          return b;
+        } else {
+          return (com.google.protobuf.ByteString) ref;
+        }
+      }
+
+      public static final int LANGUAGENAME_FIELD_NUMBER = 4;
+      private java.lang.Object languageName_;
+      /**
+       * <code>optional string languageName = 4;</code>
+       */
+      public boolean hasLanguageName() {
+        return ((bitField0_ & 0x00000008) == 0x00000008);
+      }
+      /**
+       * <code>optional string languageName = 4;</code>
+       */
+      public java.lang.String getLanguageName() {
+        java.lang.Object ref = languageName_;
+        if (ref instanceof java.lang.String) {
+          return (java.lang.String) ref;
+        } else {
+          com.google.protobuf.ByteString bs = 
+              (com.google.protobuf.ByteString) ref;
+          java.lang.String s = bs.toStringUtf8();
+          if (bs.isValidUtf8()) {
+            languageName_ = s;
+          }
+          return s;
+        }
+      }
+      /**
+       * <code>optional string languageName = 4;</code>
+       */
+      public com.google.protobuf.ByteString
+          getLanguageNameBytes() {
+        java.lang.Object ref = languageName_;
+        if (ref instanceof java.lang.String) {
+          com.google.protobuf.ByteString b = 
+              com.google.protobuf.ByteString.copyFromUtf8(
+                  (java.lang.String) ref);
+          languageName_ = b;
+          return b;
+        } else {
+          return (com.google.protobuf.ByteString) ref;
+        }
+      }
+
+      public static final int ISINHERITED_FIELD_NUMBER = 5;
+      private boolean isInherited_;
+      /**
+       * <code>optional bool isInherited = 5;</code>
+       */
+      public boolean hasIsInherited() {
+        return ((bitField0_ & 0x00000010) == 0x00000010);
+      }
+      /**
+       * <code>optional bool isInherited = 5;</code>
+       */
+      public boolean getIsInherited() {
+        return isInherited_;
+      }
+
+      public static final int PARENTKEY_FIELD_NUMBER = 6;
+      private java.lang.Object parentKey_;
+      /**
+       * <code>optional string parentKey = 6;</code>
+       */
+      public boolean hasParentKey() {
+        return ((bitField0_ & 0x00000020) == 0x00000020);
+      }
+      /**
+       * <code>optional string parentKey = 6;</code>
+       */
+      public java.lang.String getParentKey() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)